### DISCUSSION TITLE
Sandro, this is the title! Welcome to djankiserv Discussions!

### DISCUSSION THREAD

**Discussion by AntonOfTheWoods** [ankicommunity/djankiserv#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AntonOfTheWoods** [comment]
What I am really interested in, is whether this has any value over and above normal tickets. At the moment I'm vaguely positive towards the idea, but only vaguely!

EDIT: Ok, I just realised what it is - it is Github trying to steal users from Stackoverflow. Could be interesting but Stackoverflow does work quite well...

**Comment by kuklinistvan** [comment]
Wow! First of all, I have a good feeling about this. Although, shouldn't this be in the wiki repository?

**Comment by sandroden** [comment]
How is this addressing any of the addressing any of my remarks on the advantages of a forum [here](https://github.com/ankicommunity/ankicommunity.github.io/issues/6#issuecomment-740246931) or [here](https://github.com/ankicommunity/ankicommunity.github.io/issues/6#issue-758025325)?

1. It's strictly linked to a repo (not to the "project")
2. it's similar to a chat (no separate issues, no categories, no tags, no search)
3. No way to have an immediate glimp of what's going on  

The only advantages I see (over a chat):

- it's linkable
- it's separate from technical issues (but it doesn't even have a title...)

**Reply by kuklinistvan** [reply]
1. The wiki can be pointed as the place of general discussions, or we could even create an empty repo only for this purpose.
2. We have categories, although, I don't know if we have support for tags
3. It seems to have a feed, where one can read the recent issues.

As we are discussing 2 or 3 big pictures problems in parallel, I agree that a system of this sort could make our lives easier already.

**Reply by AntonOfTheWoods** [reply]
@sandroden , in my defence, I created this because I arrived on Github and had a popup say, "Activate discussions". I had no idea what it was, so obviously had to activate it and try it out!

Does it solve our problems? I think it might solve a useful problem, though not the ones you faced when trying to onboard.

I don't think a wiki is an appropriate place for discussions though. Wikis are good for document or topic focussed stuff that has long-term value, and needs to be regularly tweaked. You can usefully discuss aspects of the document/topic in a wiki but if something doesn't deserve a wiki page/section, then (I think) you should be discussing it on a wiki.

At the end of the day, nothing is perfect, and many solutions have benefits and drawbacks.

**Reply by sandroden** [reply]
> At the end of the day, nothing is perfect, and many solutions have benefits and drawbacks.

Ok, but what's the drawback of a forum? I really don't see any...

> You can usefully discuss aspects of the document/topic in a wiki but if something doesn't deserve a wiki page/section, then (I think) you should be discussing it on a wiki.

I never wanted to discuss everything on a wiki in fact... I wanted to avoid that. (maybe you missed a negation?)

**Reply by AntonOfTheWoods** [reply]
> I never wanted to discuss everything on a wiki in fact... I wanted to avoid that. (maybe you missed a negation?)

Actually that comment was directed to @kuklinistvan :-)

**Reply by kuklinistvan** [reply]
Sigh. In the wiki repository I mean. Or in an empty repository called "general discussions". Not on the page. I only proposed to have a central place, not at a specific project. Right now this is tied to the `djankiserv` repository.

**Reply by VikashKothary** [reply]
> Ok, but what's the drawback of a forum? I really don't see any...

Don't worry I've left some comments in our ticket. 😁 
JK. This is a cool feature, we'll see what happens with it. 

Also why all this hate towards wiki. I personally find traditional wiki's mildly infuriating. Or at least my experience with them. See https://xkcd.com/927/ but with wiki articles.

**Reply by sandroden** [reply]
I like wiki!!! (wikidattica forever *;-)

@kuklinistvan it's exactly the need to have discussions that do not belong to one -and only one- repository that makes it more reasonable to have a forum before going to a ticket.

**Comment by sandroden** [comment]
@AntonOfTheWoods speaking about docker vs. kubernetes said when you have a Tesla for free why should you use the motorscooter? Here the same... you're trying to use any sort of partial solution instead of a nice, comprehensive one with no disadvantages.

**Reply by AntonOfTheWoods** [reply]
Actually I really don't care, as long as everything that should go in a ticket does, and it is relatively easy to point to a discussion that has already been had, then I am fine. You can point to discussions/threads on Slack, so even that would work for me (or possibly more likely Mattermost). But discourse, discord, diswhatever are fine by me for everything except what should be in tickets. @VikashKothary wants everything except realtime chat in tickets. You appear to want everything in a forum except that which obviously should be a ticket. Again, as long as everything that really should be a ticket is one, I'm easy! :-)

Even this discussion is fine for me, though it appears that you can easily have multiple topics per discussion, which is not super-optimal

**Reply by sandroden** [reply]
> Even this discussion is fine for me, though it appears that you can easily have multiple topics per discussion, which is not super-optimal

Not sure what you mean here. Not very different from the fact that you can have several issues addressing the same problem... to be avoided clearly.

**Reply by AntonOfTheWoods** [reply]
Every tool can be abused, and every thread hijacked! Many duplicate discussions can talk place using every tool.

What is to stop us from starting a discussion on the guy in Italy who reportedly just had an argument with his wife and then spent a week walking 450kms to clear his mind? It is actually very interesting. I am a big walker myself. :-D. I am sure I could hijack a Discourse (or Discord?) thread with such interesting topics, and I am sure I could create three threads about synchronising dynamic decks. Anyway, I am officially abstaining from any vote. It's all good!

**Reply by VikashKothary** [reply]
> > Even this discussion is fine for me, though it appears that you can easily have multiple topics per discussion, which is not super-optimal
> 
> Not sure what you mean here. Not very different from the fact that you can have several issues addressing the same problem... to be avoided clearly.

So what I like is that we're on the same page about the problem. 

But just disagree on the solution. I personally want to have one `backend` repo and this should minimise duplicate issues. That can only happen when this repo has all the features from the `anki-sync-server`. That's my current priority.

**Reply by VikashKothary** [reply]
> Every tool can be abused, and every thread hijacked! Many duplicate discussions can talk place using every tool.
> 
> What is to stop us from starting a discussion on the guy in Italy who reportedly just had an argument with his wife and then spent a week walking 450kms to clear his mind? It is actually very interesting. I am a big walker myself. :-D. I am sure I could hijack a Discourse (or Discord?) thread with such interesting topics, and I am sure I could create three threads about synchronising dynamic decks. Anyway, I am officially abstaining from any vote. It's all good!

That reminded me of this: https://github.com/palantir/tslint/issues/4141. People just hijacking their Github Issues to bully the company.
